src/formatter/lib/resolveLinks.ts

```diff
@@ -83,25 +83,21 @@ function _preProcessLinksInTextBlock(sText: string, ui5Url: string, ui5Version:
 		}
 
 		// If the link has a protocol, do not modify, but open in a new window
-		if (/:\/\//.test(sTarget)) {
```

Comment:
Changes in this file should rather be applied within the actual source file, which is located at https://github.com/SAP/openui5/blob/master/lib/jsdoc/transformApiJson.js

For that reason, I wonder whether we should rather disable the rules for this file?

Reply:
Done